browsers/README.md

```diff
@@ -41,7 +41,7 @@
 [cypress/browsers:node16.14.0-chrome99-ff97](./node16.14.0-chrome99-ff97) | `cypress/base:16.14.0-slim` | `99.0.4844.51` | `97.0.1` | `🚫` 
 [cypress/browsers:node16.14.2-slim-chrome100-ff99-edge](./node16.14.2-slim-chrome100-ff99-edge) | `cypress/base:16.14.2` | `100.0.4896.88` | `99.0.1` | `--edge` 
 [cypress/browsers:node14.17.6-slim-chrome100-ff99-edge](./node14.17.6-slim-chrome100-ff99-edge) | `cypress/base:14.17.6` | `100.0.4896.88` | `99.0.1` | `--edge`
-[cypress/browsers:node14.19.0-chrome100-ff99-edge](./node14.19.0-chrome100-ff99-edge) | `cypress/base:14.19.0` | `100.0.4896.88` | `99.0.1` | `--edge` 
+[cypress/browsers:node14.19.0-chrome100-ff99-edge](./node14.19.0-chrome100-ff99-edge) | `cypress/base:14.19.0` | `100.0.4896.88` | `99.0.1` | `--edge`[cypress/browsers:node16.14.2-slim-chrome103-ff99](./node16.14.2-slim-chrome103-ff99) | `cypress/base:16.14.2` | `103.0.5060.53` | `99.0.1` | `🚫` 
```

Comment:
A line break is missing here too. Neither the readme or changelog render the 16.14.2 version as a result.